### ISSUE TITLE
Error with import function on beta

### ISSUE THREAD

**Issue by nchun2** [dondi/GRNsight#1103]
I was trying to import a small-scale network to be able to put a graphic of the sensitivity assay on my poster for URS, but GRNsight was giving me the following errors
![Screenshot (146)](https://github.com/dondi/GRNsight/assets/122498296/864f67ea-42dc-4a55-91ee-19a06f74d4c2)
The following file is the excel that I was using as an input 
[Copy of DB5_derived_sample_network_output.xlsx](https://github.com/dondi/GRNsight/files/14652884/Copy.of.DB5_derived_sample_network_output.xlsx)

**Comment by kdahlquist** [comment]
I think that this is a problem with Excel having hidden holdover information in the file from having edited a previous workbook.  I was able to fix the file and get it to load by deleting the empty rows at the bottom of the production_rates and degradation_rates sheets.

I deleted the whole rows as opposed to just clearing the contents of the cells.

When I then loaded it into GRNsight, the graph displayed, although there was a warning that there was no value for the "penalty" in the optimization_diagnostics sheet.  I also saw that the "iteration count" was 0.  Was this a direct GRNmap output?  If so, something strange is going on with GRNmap.
